src/interfaces/ISablierV2Lockup.sol

```diff
@@ -134,6 +134,9 @@ interface ISablierV2Lockup is
     /// @notice Counter for stream ids, used in the create functions.
     function nextStreamId() external view returns (uint256);
 
+    /// @notice Contract that generates the non-fungible token URI.
```

Comment:
How about this?

```solidity
/// @notice Retrieves the address of NFT descriptor contract that generates the non-fungible token URI.
```

Reply:
it is better like it is now

we usually use "Retrieves" word for getters that implement the function within a scope. here it is not the case, as it is implemented automatically by the language when:

```solidity
ISablierV2NFTDescriptor public override nftDescriptor;
```